### PR TITLE
Gnome 44.1 Builds (some)

### PR DESCRIPTION
**Pull request by satmandu**
Builds properly:
- [x] `x86_64` (tested)
- [x] `i686` (some builds removed intentionally)
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=gnome441 CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/gnutls.rb [hunk 1]
```diff
@@ -3,41 +3,49 @@
...
+  depends_on 'zlibpkg' => :build
+  depends_on 'zstd' # R
+
+  no_env_options
 
   def self.prebuild
     # Use IPv4 fallback if default connection fails.
     system "#{CREW_PREFIX}/sbin/unbound-anchor -a '#{CREW_PREFIX}/etc/unbound/root.key' || #{CREW_PREFIX}/sbin/unbound-anchor -4 -a '#{CREW_PREFIX}/etc/unbound/root.key'"
+    # Rebuild ca-certificates.
+    system "#{CREW_PREFIX}/bin/update-ca-certificates --fresh --certsconf #{CREW_PREFIX}/etc/ca-certificates.conf"
```
Comment (uberhacker) [inline packages/gnutls.rb:48]: Could the test failures be due to the way the certificates are generated?

File: packages/wayland.rb [hunk 2]
```diff
@@ -6,18 +6,20 @@ class Wayland < Package
   @_ver = '1.22.0'
   version @_ver
   license 'MIT'
-  compatibility 'x86_64 aarch64 armv7l'
+  compatibility 'all'
```
Comment (uberhacker) [inline packages/wayland.rb:9]: So we haven't given up on i686 yet? :smile:

Comment (satmandu) [reply packages/wayland.rb:9]: It seems to build just fine, so I leave it.

File: packages/vala.rb [hunk 3]
```diff
@@ -3,32 +3,33 @@
...
+    aarch64: '4987cd9bdffb4056e046e81e11600bd7eb38085ce76fc87822b20be2b31aeba6',
+     armv7l: '4987cd9bdffb4056e046e81e11600bd7eb38085ce76fc87822b20be2b31aeba6',
+       i686: '794bcb3f8b6d1f2e6cbb6b4ad0029dfabbc00a2c830df13c5939d06c6afb93ab',
+     x86_64: '55f786136cccd4bc28ed2f28056a09c143fe5acd2d1af6d08fe80bd228e725a2'
   })
 
   depends_on 'autoconf_archive' => :build
   depends_on 'autoconf213' => :build
-  depends_on 'graphviz'
+  depends_on 'graphviz' unless ARCH == 'i686'
```
[the remark this answers is not in the document]
Comment (satmandu) [reply packages/vala.rb]: done.

File: packages/gnutls.rb (hunk 1 above)
Comment (satmandu) [reply packages/gnutls.rb:48]: I don't think so at all. I put that line there after I noticed this error, as part of making sure that there were updated certs, which some builds (of other packages) will complain about.

**Pull request merged**